db/seeds.rb

```diff
@@ -1,8 +1,9 @@
-admin_role = Role.find_or_create_by(name: "admin")
 
 grove = Grove.create!(name: "Aspen")
 Grove.create!(name: "Fir")
 Grove.create!(name: "Pine")
+Role.create!(name: "teacher")
```

Comment:
makes sense, got it. Will just have to take account the empty string that blank sends over. But not a big deal at all. That approach makes much more sense.